katalog/configs/ingress-nginx/flow.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: logging.banzaicloud.io/v1beta1
+kind: Flow
+metadata:
+  name: ingress-nginx
+spec:
+  filters:
+    - dedot:
+        de_dot_separator: "_"
+        de_dot_nested: true
+    - parser:
+        remove_key_name_field: true
+        parse:
+          type: nginx
+  match:
+    - select:
+        labels:
+          app: ingress-nginx
+  localOutputRefs:
+    - ingress-nginx
+
+# Alternatively:
```

Comment:
can we explain a little when should the user choose the commented version instead of the default one?

Reply:
Done ✅